app/routes.js

```diff
@@ -7,3 +7,17 @@ const govukPrototypeKit = require('govuk-prototype-kit')
 const router = govukPrototypeKit.requests.setupRouter()
 
 // Add your routes here
+
+router.post('/site-service', function(request, response) {
+   //  console.log(request.body)
```

Comment:
I dont want to be too pedantic since this isjust the prototype repo but can we get rid of these comments?

Also in general, I feel like it would be good for the team to be given a demo about how best to utilise GDS prototyping repo and what a good workflow is, because, like we discussed on our previous call, if Im going to review your PRs just to blindly approve them then it defeats the purpose of the PR.